apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/CollectionFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,28 @@
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.fun1
+import ch.tutteli.atrium.specs.name
+import ch.tutteli.atrium.specs.notImplemented
+import ch.tutteli.atrium.specs.property
+
+class CollectionFeatureAssertionsSpec : ch.tutteli.atrium.specs.integration.CollectionFeatureAssertionsSpec(
+    property<Collection<String>, Int>(Expect<Collection<String>>::size).name to Expect<Collection<String>>::size,
```

Comment:
since you don't use a companion function you can use directly
```suggestion
    property<Collection<String>, Int>(Expect<Collection<String>>::size),
```

Reply:
I changed  that, will be up in the next push